src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -276,51 +270,6 @@
   <data name="InvalidHeaderName" xml:space="preserve">
     <value>An invalid character was found in header name.</value>
   </data>
-  <data name="net_cookie_attribute" xml:space="preserve">
-    <value>The '{0}'='{1}' part of the cookie is invalid.</value>
-  </data>
-  <data name="net_http_unix_invalid_credential" xml:space="preserve">
-    <value>The libcurl library in use ({0}) does not support different credentials for different authentication schemes.</value>
-  </data>
-  <data name="net_http_unix_https_support_unavailable_libcurl" xml:space="preserve">
-    <value>The libcurl library in use ({0}) does not support HTTPS.</value>
-  </data>
-  <data name="ArgumentOutOfRange_FileLengthTooBig" xml:space="preserve">
-    <value>Specified file length was too large for the file system.</value>
```

Comment:
A bunch of these aren't actually dead and are used by various builds, e.g. the unix build.

Reply:
Yes, I noticed it. Just a moment. :)